src/test/java/org/opentripplanner/middleware/docs/SwaggerTest.java

```diff
@@ -54,312 +36,16 @@ public void updateSwaggerDocs() throws IOException, InterruptedException {
     }
 
     /**
-     * Generate enhanced Swagger docs for use with AWS API Gateway. This handles things like adding auth headers and key
-     * parameters used with API Gateway as well as injecting info needed to make CORS work.
+     * Convenience test to generate the docs.
```

Comment:
Does this need to be a test? It doesn't seem to be testing anything...

Reply:
Hehe no, I'm still working on this.